internal/service/tesla_fleet_api_service.go

```diff
@@ -80,21 +80,23 @@ type fleetStatusResponse struct {
 	KeyPairedVINs []string `json:"key_paired_vins"`
 	UnpairedVINs  []string `json:"unpaired_vins"`
 	VehicleInfo   map[string]struct {
-		FirmwareVersion                string `json:"firmware_version"`
-		VehicleCommandProtocolRequired bool   `json:"vehicle_command_protocol_required"`
-		DiscountedDeviceData           bool   `json:"discounted_device_data"`
-		FleetTelemetryVersion          string `json:"fleet_telemetry_version"`
-		TotalNumberOfKeys              int    `json:"total_number_of_keys"`
+		FirmwareVersion                    string `json:"firmware_version"`
+		VehicleCommandProtocolRequired     bool   `json:"vehicle_command_protocol_required"`
+		DiscountedDeviceData               bool   `json:"discounted_device_data"`
+		FleetTelemetryVersion              string `json:"fleet_telemetry_version"`
+		TotalNumberOfKeys                  int    `json:"total_number_of_keys"`
+		SafetyScreenStreamingToggleEnabled *bool  `json:"safety_screen_streaming_toggle_enabled"`
```

Comment:
Adding new filed safety_screen_streaming_toggle_enabled, which could be nil, true or false